solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -867,6 +869,8 @@ public static class Builder
 
     protected Long keyStoreReloadIntervalSecs;
 
+    private List<HttpListenerFactory> listenerFactory;
```

Comment:
There's no setter for this.  Which is suspicious... like do we even need this in this PR?  If you add a setter for it, you can use the setter in UpdateHandler.